src/mappings/constants.ts

```diff
@@ -23,3 +23,6 @@ export const VALUE_KEY = b64encode("value");
 export const STORE_KEY = b64encode("store");
 export const VSTORAGE_VALUE = b64encode("vstorage");
 export const KEY_KEY = b64encode("key");
+export const STORE_NAME_KEY = b64encode("store_name");
+export const SUBKEY_KEY = b64encode("store_subkey");
+export const UNPROVED_VALUE_KEY = b64encode("unproved_value");
```

Comment:
`SUBKEY_KEY` ......... KEY KEY :D Perhaps, `SUB_KEY`? 

I think this is cleaner:
```
export const STORE_NAME = b64encode("store_name");
export const SUB_KEY = b64encode("store_subkey");
export const UNPROVED_VALUE = b64encode("unproved_value");
```

Line 25 too :D

Reply:
the repetition is there intentionally to indicate the actual name of the key. for keys that end with "key" the name might look strange but i think we should keep the same pattern among all consts to avoid confusion